Simplenote/PreferencesViewController.swift

```diff
@@ -0,0 +1,314 @@
+import Cocoa
+
+class PreferencesViewController: NSViewController {
+    let simperium = SimplenoteAppDelegate.shared().simperium
```

Comment:
This one shouldn't be a constant (perhaps a dynamic property?).

We're not doing it, but in theory, the `Simperium` instance could be swapped, and we could be left there with an old reference.

Plus: `private`